### PR TITLE
Adding docker-compose config file + instructions

### PR DESCRIPTION
**Pull request by theute**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # notifications-gw project
 
+To run the application or the testsuite you'll need Kafka running on port 9092.
```
Comment (josejulio) [inline README.adoc:3]: testsuite doesn't need to have kafka running. The tests already take care of starting one.

Comment (josejulio) [inline README.adoc:3]:
```suggestion
To run the application you'll need Kafka running on port 9092.
```

Comment (theute) [reply README.adoc]: True

**Review by josejulio — APPROVED** [review]
LGTM

**Pull request merged**